package.json

```diff
@@ -63,12 +64,14 @@
     "xml-escape": "~1.0.0"
   },
   "devDependencies": {
+    "@ljharb/eslint-config": "^1.0.4",
```

Comment:
Why do we have to add these dependencies rather we can you use `proxyquire` to fake these dependencies. proxtquire has been used inside `config` unit tests.